agent/doctor/docker_runtime_healthcheck.go

```diff
@@ -45,12 +47,13 @@ type dockerRuntimeHealthcheck struct {
 }
 
 func NewDockerRuntimeHealthcheck(client dockerapi.DockerClient) *dockerRuntimeHealthcheck {
-	nowTime := time.Now()
+	nowTime := timeNow()
```

Comment:
What happens if the healthcheck request is not sent before the health message is sent from the Agent to the Control plane?

Reply:
Yes. What will be the value set for `lastUpdatedTime` in that scenario?

Reply:
Synced offline